go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/square/go-jose/v3
 go 1.12
 
 require (
+	github.com/google/go-cmp v0.4.0
```

Comment:
This is more for output readability on the test but happy to drop it if you'd rather not add the dependency

Reply:
seems fine to me

Reply:
especially that Go modules knows when an import is used in tests, and even though this dep is present in go.mod, it will not bloat the actual code.